website/src/app/[lang]/[region]/(website)/journal/StoryblokApi.ts

```
@@ -0,0 +1,254 @@
+	const dateObject = DateTime.fromISO(date).setLocale(lang);
+	return dateObject.isValid ? dateObject.toFormat('MMMM dd, yyyy') : '';
+}
+
+// During the development of Storyblok features or writing of new content, it is useful to use the draft version of the content.
+function addVersionParameter(properties: ISbStoriesParams): ISbStoriesParams {
+	return { ...properties, version: draftMode().isEnabled ? 'draft' : 'published' };
+}
+
+// Based on official documentation: https://www.storyblok.com/faq/image-dimensions-assets-js
+// format example: a.storyblok.com/f/51376/664x488/f4f9d1769c/visual-editor-features.jpg
+export function getDimensionsFromStoryblokImageUrl(url: String): { width: number; height: number } {
```

Biome (1.9.4)

[error] Don't use 'String' as a type.

Use lowercase primitives for consistency.
Safe fix: Use 'string' instead


(lint/complexity/noBannedTypes)